internal/pkg/aws/ec2/ec2.go

```diff
@@ -377,3 +364,76 @@ func toEC2Filter(filters []Filter) []*ec2.Filter {
 	}
 	return ec2Filter
 }
+
+type routeTable ec2.RouteTable
+
+// IsMain returns true if the route table is the default route table for the VPC.
+// If a subnet is not associated with a particular route table, then it will default to the main route table.
+func (rt *routeTable) IsMain() bool {
+	for _, association := range rt.Associations {
+		if aws.BoolValue(association.Main) {
```

Comment:
hmm I wonder why `Main` is an attribute of `association` but not `routeTable`. Sounds to me the latter makes much more sense 😅

Reply:
That's a great point 😅 I have no idea why it's like that